ansible/roles/jenkins.yml

```
@@ -0,0 +1,6 @@
+---
+- name: Jenkins
+  hosts: jenkins
+  become: true
+  roles:
+    - jenkins
```

yaml[new-line-at-end-of-file]: No new line character at the end of file
```suggestion
    - jenkins

```